pennychat/tests/api/test_penny_chat.py

```diff
@@ -38,13 +77,34 @@ def test_penny_chat_participants_list(test_chats_1):
         assert int(user_id) in [participant['user']['id'] for participant in chat['participants']]
 
 
+@pytest.mark.django_db
+def test_penny_chat_participants_list__other_content():
+    objects = setup_test_chats()
+    chat_private_1, chat_private_2 = objects['penny_chats']
+    user1, user2, user3 = objects['users']
+    soc_prof1, soc_prof2, soc_prof3 = objects['social_profiles']
+    client = APIClient()
+    token = Token.objects.create(user=user3)
+    client.credentials(HTTP_AUTHORIZATION='Token ' + token.key)
+    user_id = user2.id
+    response = client.get(f'/api/chats/?participants__user_id={user_id}')
+    assert response.status_code == 200
+    assert response.data['count'] == 0
+    chats = response.data['results']
+    for chat in chats:
+        assert int(user_id) in [participant['user']['id'] for participant in chat['participants']]
+
+
 @pytest.mark.django_db
 def test_penny_chat__upcoming_or_popular(test_chats_2):
```

Comment:
Similar for this test:

As a logged in user who looks at the /chats/ endpoint:
* I should be able to see all public chats
* I should be able to see all private chats where I was a participant
* I should NOT be able to see any private chats where I was not a participant